systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleResource.java

```diff
@@ -56,38 +56,42 @@ public boolean waitForReadiness(Deployment resource) {
     }
 
     public static DeploymentBuilder clusterOperator(String namespace, long operationTimeout) {
-        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespace, operationTimeout, Constants.RECONCILIATION_INTERVAL);
+        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespace, operationTimeout, Constants.RECONCILIATION_INTERVAL, null);
     }
 
     public static DeploymentBuilder clusterOperator(String namespace, String namespaceEnv, long operationTimeout, long reconciliationInterval) {
-        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, operationTimeout, reconciliationInterval);
+        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, operationTimeout, reconciliationInterval, null);
     }
 
     public static DeploymentBuilder clusterOperator(String namespace, String namespaceEnv, long reconciliationInterval) {
-        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, Constants.CO_OPERATION_TIMEOUT_DEFAULT, reconciliationInterval);
+        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, Constants.CO_OPERATION_TIMEOUT_DEFAULT, reconciliationInterval, null);
     }
 
     public static DeploymentBuilder clusterOperator(String namespace, String namespaceEnv) {
-        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL);
+        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespaceEnv, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL, null);
     }
 
     public static DeploymentBuilder clusterOperator(String namespace, long operationTimeout, long reconciliationInterval) {
-        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespace, operationTimeout, reconciliationInterval);
+        return defaultClusterOperator(Constants.STRIMZI_DEPLOYMENT_NAME, namespace, namespace, operationTimeout, reconciliationInterval, null);
     }
 
     public static DeploymentBuilder clusterOperator(String name, String namespace, String namespaceEnv, long operationTimeout, long reconciliationInterval) {
-        return defaultClusterOperator(name, namespace, namespaceEnv, operationTimeout, reconciliationInterval);
+        return defaultClusterOperator(name, namespace, namespaceEnv, operationTimeout, reconciliationInterval, null);
     }
```

Comment:
This is not really a good sign of clean code...When you have more than 5-6 overloaded methods it's better to use the Builder pattern. I know that these methods have been there before, but if they are extended, the alternatives I have mentioned should be considered. Like I have done in my current PR https://github.com/strimzi/strimzi-kafka-operator/pull/5105.